scripts/notify-suppliers-of-new-fw-cq-answers.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
```

Comment:
Ah - can we make this file executable, to be consistent with our other scripts?

Reply:
ah crap

Reply:
ok fixed

Reply:
Great - merge away!